src/components/views/menu/menu.jsx

```diff
@@ -78,6 +86,10 @@ export default class Menu extends React.Component {
          */
         pointing: React.PropTypes.bool,
         /**
+         * A menu can be formatted to float left
+         */
+        left: React.PropTypes.bool,
+        /**
          * A menu can be formatted to float right
```

Comment:
There is already floated prop, but it turns class name into left floated and right floated. Not sure if left floated/right floated className makes sense, may be fix it then?
i.e ``` classes[this.props.floated] = !!this.props.floated ?